.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
remove this file too.
please read the .gitignore file again, and remove all files that are in the list of .gitignore.